include/interfaces/non_conservative/navier_stokes.h

```diff
@@ -256,6 +275,12 @@ NavierStokes<dim>::
 parse_parameters_call_back ()
 {
   NonConservativeInterface<dim,dim,dim+1, NavierStokes<dim> >::parse_parameters_call_back();
+
+  this->prm.leave_subsection();
```

Comment:
'''
You can't askInput grid file name> you have not y for entry <Input grid file name> you have not yet det declared 
'''